integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java

```diff
@@ -1761,38 +1962,53 @@ private static void deployApplication(String namespace, String domainUid, String
     assertEquals("202", result.stdout(), "Deployment didn't return HTTP status code 202");
   }
 
-  private void verifyAdminServerAccess(String domainUid) {
-    String consoleUrl = new StringBuffer()
-        .append("http://")
-        .append(K8S_NODEPORT_HOST)
+  private void verifyAdminServerAccess(boolean isTLS,
+                                       int lbNodePort,
+                                       boolean isHostRouting,
+                                       String ingressHostName,
+                                       String pathLocation) {
+
+    StringBuffer consoleUrl = new StringBuffer();
+    if (isTLS) {
+      consoleUrl.append("https://");
+    } else {
+      consoleUrl.append("http://");
+    }
+    consoleUrl.append(K8S_NODEPORT_HOST)
```

Comment:
As we discussed, can u pass extra headers in curl command while accessing the console as follows ...
   -H "WL-Proxy-Client-IP: 120.1.1.0" -H "WL-Proxy-SSL: false"  and make sure that this header does not appear in  adminserver log. 

To get this header info you need to add the following DebugFlag on server startup args 

-Dweblogic.http.isWLProxyHeadersAccessible=true -Dweblogic.debug.DebugHttp=true -Dweblogic.StdoutDebugEnabled=true

Reply:
fixed